container-images/rocm/Containerfile

```diff
@@ -16,23 +16,37 @@ RUN curl --retry 8 --retry-all-errors -o \
       cat /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official
 RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official
 
-RUN dnf install -y rocm-dev hipblas-devel rocblas-devel && \
```

Comment:
How does this work without the dnf install's for the ROCm libs hmmm

Reply:
Apologies and good catch! I realized this just after making the PR!  I decided to make a change manually to keep as close to main as possible but accidentally deleted those lines.